src/assets/logos/Nutrien-Logo.wine.svg

```diff
@@ -0,0 +1 @@
3 2 0 3.9.2 5.8.6 1.8.3 3.4 1.1 4.9 2.2 1.4 1.1 2.4 2.7 2.8 4.5.5 2.5.3 5.1-.5 7.5l-9.5 36.4c-.7 2.9-1.3 5.4-1.7 7.7-.4 2-.4 4-.1 5.9.4 1.7 1.4 3.1 2.9 3.9 1.6.9 4 1.4 7.2 1.4 4.4.1 8.7-.9 12.6-2.8 3.8-1.9 6.1-4.3 6.8-7.1l15.4-59.3c1-.2 2.1-.4 3.1-.5 1.9-.2 3.8-.3 5.7-.3 2 0 3.9.2 5.8.7 1.8.4 3.4 1.2 4.8 2.4 1.4 1.3 2.3 3 2.7 4.8.4 2.6.3 5.2-.5 7.7z" class="st2"/><path id="path5925" d="M226.6 117.4c-.2-2.6-.1-5.2.4-7.8.5-2.7 1-5.5 1.8-8.2l22.4-86.2 2.9-.4c.9-.1 1.9-.2 2.9-.3 1-.1 2-.1 3-.1 2 0 3.9.2 5.8.6 1.8.3 3.4 1.1 4.8 2.2 1.4 1.2 2.3 2.8 2.7 4.6.5 2.5.3 5.1-.5 7.5l-3.4 13 26.2.1-4.9 19-26.2-.1-10.3 39.7c-2.3 8.7.2 13 7.3 13 1.8 0 3.5-.3 5.2-.8 1.6-.5 3.2-1.1 4.8-1.8 1.3-.6 2.6-1.3 3.8-2.1 1.1-.7 1.8-1.3 2.3-1.7 1 2.3 1.8 4.7 2.5 7.1.5 1.8.5 3.7 0 5.6-.6 1.9-1.7 3.5-3.2 4.7-2 1.7-4.2 3.2-6.6 4.3-3 1.4-6.2 2.5-9.4 3.1-3.7.8-7.5 1.2-11.3 1.2-7.8 0-13.5-1.5-17-4.5-3.6-2.9-5.8-7.1-6-11.7" class="st2"/><path id="path5927" d="M334.4 51.9c1.3-1.3 2.6-2.6 4-3.7 3.6-2.9 7.7-5.1 12.2-6.5 2.4-.7 4.8-1.1 7.3-1.1 1.6 0 3.1.1 4.7.4 1.5.2 3 .7 4.3 1.4 2.7 1.3 3.9 4.4 2.9 7.2-1 3.3-2.5 6.4-4.5 9.3-2 3.1-4.2 6.2-6.5 9.1-1.5-1.6-3.3-3-5.3-4-2.2-1-4.7-1.5-7.1-1.4-3.8 0-7.5 1.1-10.6 3.3-2.9 1.8-5 4.7-6 8L314.9 131h-25.4l22.9-87.9 2.3-.4c.8-.1 1.7-.2 2.7-.3 1-.1 2.2-.1 3.6-.1 3.4-.1 6.8.7 9.8 2.3 2.5 1.6 3.9 4.4 3.6 7.3" class="st2"/><path id="path5929" d="M385.7 132.3c-1.1.3-2.3.5-3.5.6-1.8.1-3.7.2-5.6.2-4.9 0-8.5-1-11.1-3.1-2.5-2.1-3-6.2-1.4-12.4l19.4-74.3c1-.2 2.1-.4 3.1-.5 1.9-.2 3.8-.3 5.7-.3 2 0 3.9.2 5.8.6 1.8.3 3.4 1.1 4.9 2.2 1.4 1.1 2.4 2.7 2.8 4.5.5 2.5.3 5.1-.5 7.5zm31.1-118.7c-1 3.9-3.3 7.3-6.5 9.6-3 2.4-6.7 3.7-10.6 3.7-3.9 0-6.8-1.2-8.8-3.7-2-2.5-2.4-5.7-1.4-9.7 1-3.9 3.3-7.3 6.5-9.8 3-2.4 6.8-3.8 10.7-3.8 3.3-.2 6.5 1.2 8.6 3.8 2 2.7 2.5 5.9 1.5 9.9" class="st2"/><path id="path5931" d="M454.3 115.6c2.9 0 5.7-.3 8.5-1 2.4-.6 4.7-1.5 7-2.5 2-.9 3.9-2 5.7-3.2 1.7-1.2 3.2-2.3 4.6-3.2 1.2 2.3 2.1 4.7 2.8 7.2.8 2.3 1 4.8.6 7.2-.7 2.7-2.9 5.2-6.6 7.4-3.9 2.3-8.1 4-12.5 5.1-5.7 1.5-11.6 2.2-17.5 2.1-5.6 0-11.3-.7-16.7-2.3-5-1.3-9.6-4.1-13.1-7.8s-5.7-8.7-6.9-15c-1.1-6.3-.4-14.2 2.1-23.7 1.8-7.2 4.8-14 8.9-20.2 3.7-5.4 8.2-10.1 13.4-14 4.8-3.6 10.2-6.3 15.9-8.1 5.3-1.7 10.8-2.5 16.4-2.6 7 0 12.6 1.1 16.9 3.3 4 1.9 7.3 5 9.6 8.8 2.1 3.8 3.3 8.2 3.4 12.6.1 5-.5 10-1.8 14.9l-.7 2.6c-.3 1-.6 2-.9 3.2l-1 3.3c-.3.9-.6 1.7-.9 2.5l-54.9-.1c-1.3 6.8-.6 12.5 1.9 17 2.4 4.1 7.7 6.4 15.8 6.5m18.2-37.8c1.5-5.9 1.7-10.8.3-14.5-1.3-3.7-5-5.6-11-5.6-5.1-.1-10 1.8-13.7 5.3-3.9 3.5-6.7 8.4-8.3 14.8z" class="st2"/><path id="path5933" d="M524.1 131.6l-25.6-.1 22.9-87.9 2.3-.4c.8-.1 1.7-.2 2.7-.3 1-.1 2.2-.1 3.6-.1 3.4-.1 6.8.6 9.8 2.1 2.6 1.4 3.7 3.9 3.5 7.5 3.5-3.3 7.4-5.9 11.8-7.9 5-2.4 11.2-3.6 18.7-3.6 8.8 0 14.9 1.8 18.3 5.3 3.3 3.5 4.9 8.1 4.6 13.8-.2 3.1-.6 6.2-1.2 9.3-.7 3.3-1.4 6.6-2.3 9.8l-13.6 52.4-25.4-.1 13.3-51.3c.7-2.7 1.3-5.3 1.7-7.6.4-2 .4-4 .1-6-.4-1.7-1.4-3.1-2.9-3.9-1.6-.9-4-1.4-7.2-1.4-4.4-.1-8.7.9-12.6 2.8-3.8 1.9-6.1 4.3-6.8 7.1z" class="st2"/></g></svg>
```

Comment:
This is open source now, so it should probably be something generic, or a Nearform logo?